roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -34,7 +34,7 @@ def run(self, tmp, task_vars):
             }
 
         # attempt to get the docker info hash from the API
-        info = self.execute_module("docker_info", {}, task_vars)
+        info = self.execute_module("docker_info", {}, task_vars=task_vars)
```

Comment:
As I said before, I don't know when Ansible passes to our action plugin `tmp != None`.
I think the correct behavior here and the other similar places would be to forward whatever value of `tmp` `run` has received.